.travis.yml

```diff
@@ -4,11 +4,16 @@ scala:
 - 2.10.6
 
 before_install:
- - export PATH=${PATH}:./vendor/bundle
+- export PATH=${PATH}:./vendor/bundle
 
 install:
-  - gem install jekyll -v 2.5
+- gem install jekyll -v 2.5
```

Comment:
It's limited by the ruby version installed in Travis. It's just for CI purpose so we shouldn't care.